api/v1alpha1/etcdcluster_types.go

```diff
@@ -16,13 +16,20 @@ type EtcdClusterSpec struct {
 	Storage *EtcdPeerStorage `json:"storage,omitempty"`
 }
 
+type EtcdMember struct {
+	Name string `json:"name"`
+	ID   string `json:"ID"`
```

Comment:
add some comments to these fields please, even if they're trivial

Reply:
I think maybe the json spelling of ID should be `id`. See: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#naming-conventions and the `metadata.uid` field, for example.

Reply:
Corrected JSON spelling to `id`. I left the Go spelling as `ID` and not `Id` as `ID` is an initialism.